### PR TITLE
Add processes endpoint

### PR DESCRIPTION
**Pull request by NinadBhat**

### CODE REVIEW

File: aiida_restapi/models.py [hunk 1]
```diff
@@ -129,3 +130,26 @@ class Group(AiidaModel):
...
     description: Optional[str] = Field(description="Short description of the group.")
+
+
+class Process(AiidaModel):
+    """AiiDA Process Model"""
+
+    _orm_entity = orm.ProcessNode
+
+    id: Optional[int] = Field(description="Unique id (pk)")
+    uuid: Optional[UUID] = Field(description="Unique uuid")
```
Comment (ltalirz) [inline aiida_restapi/models.py:141]: UUID stands for "universally unique identifier", i.e. "unique" is already part of the abbreviation

File: aiida_restapi/routers/process.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""Declaration of FastAPI application."""
```
Comment (ltalirz) [inline aiida_restapi/routers/process.py:2]:
```suggestion
"""Declaration of FastAPI router for processes."""
```

File: conftest.py [hunk 3]
```diff
@@ -53,6 +55,44 @@ def default_computers():
     return [comp_1.id, comp_2.id]
 
 
+@pytest.fixture(scope="function")
+def default_process():
```
Comment (ltalirz) [inline conftest.py:59]:
```suggestion
def example_processes():
```

File: conftest.py [hunk 4]
```diff
@@ -53,6 +55,44 @@ def default_computers():
...
+    """Populate database with some process"""
```
Comment (ltalirz) [inline conftest.py:60]:
```suggestion
    """Populate database with some processes"""
```

**Review by ltalirz — COMMENTED** [review]
hi @NinadBhat , fine to continue along these lines

if you have any specific questions let us know

File: .pre-commit-config.yaml [hunk 5]
```diff
@@ -58,6 +58,7 @@ repos:
       - passlib
       - pytest~=3.6,<5.0.0
       - sphinx<4
+      - importlib_metadata~=4.6.1
```
Comment (chrisjsewell) [inline .pre-commit-config.yaml:61]:
```suggestion
      - importlib_metadata~=4.3
```

to be inline with https://github.com/aiidateam/aiida-core/pull/5058

and in setup.json

**Review by chrisjsewell — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml (hunk 5 above)
Comment (NinadBhat) [reply .pre-commit-config.yaml]: @chrisjsewell  I have made the changes

**Review by chrisjsewell — APPROVED** [review]
Looks good thaks 😄

**Pull request merged**